src/search-schema/data/definitions/enums/assay_types.yaml

```diff
@@ -201,20 +235,21 @@ salmon_rnaseq_snareseq:
     description: snRNA-seq (SNARE-seq2) [Salmon]
     alt-names: []
     primary: false
+    contains-pii: false
     vitessce-hints: ['is_sc', 'rna']
 
 sc_atac_seq_snare:
-#SNAREseq_snapATAC:
```

Comment:
I don't like this key for the same reason: no indication in the key that it is not raw atac-seq data, which should be protected.

Reply:
The 'primary: true' flag is supposed to distinguish primary datasets.  I agree that a lot of the enums (like sc_atac_seq_snare in this case) are poorly chosen.  Once we have this mechanism in place we can relegate those to being 'alt-names', with better enums for the primary keys.

Reply:
OK, sounds good.